button-2/editor-blocks.css

```diff
@@ -140,8 +140,8 @@ Description: Gutenberg Block Editor Styles
 
 /* List styles */
 
-.edit-post-visual-editor ul:not(.wp-block-gallery) li,
-.editor-block-list__block ul:not(.wp-block-gallery) li,
+.edit-post-visual-editor figure:not(.wp-block-gallery) ul li,
```

Comment:
This causes a standard `ul li` from being skipped. I think we can use the `gallery-item` class to keep the styling for non-gallery lists the same:

```
.edit-post-visual-editor ul li:not(.gallery-item),
.editor-block-list__block ul li:not(.gallery-item),
...
```

Reply:
Mentioned the wrong class name in my original comment. Updated to `.blocks-gallery-item`.

Reply:
@michaeldcain I've updated the PR according to your feedback in https://github.com/Automattic/themes/pull/1729/commits/00dde082833a2601afa7d4473d884077827648b4